Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -593,11 +676,41 @@ class FlowDefinitionPanel extends React.Component {
             isSaving: true,
             showMessageBar: false,
             messageBarIsError: false,
-            inProgessDialogMessage: 'Deployment in progress'
+            inProgessDialogMessage: 'Save in progress'
         });
 
         this.props
             .onSaveFlow(this.props.flow)
+            .then(name => {
+                this.setState({
+                    isSaving: false,
+                    showMessageBar: true,
+                    messageBarIsError: false,
+                    messageBarStatement: 'Flow definition is saved. If there are any scheduled batch jobs in thie flow, they will be created and started by the scheduler.'
```

Comment:
I recommend you provide different message bar statements based on first checking if the flow input type is batch versus streaming. Its best to provide messages that are contextual. i.e. I find it odd that we mention about batch even for the streaming case. Lets not make our customers think since we can provide more contextual messages. BTW, this above is spelled incorrectly (i.e. thie flow)

Reply:
fixed. show this message only if it is a batch mode.